tower/src/buffer/worker.rs

```diff
@@ -70,10 +70,11 @@ where
     ///
     /// If a `Message` is returned, the `bool` is true if this is the first time we received this
     /// message, and false otherwise (i.e., we tried to forward it to the backing service before).
+    #[allow(clippy::type_complexity)]
```

Comment:
i wonder if we can break this up a little with a type error?

Reply:
I don't follow